src/workflow/UberRefProvider.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+final class UberRefProvider {
```

Comment:
Theoretically this file should be in some other folder.

Reply:
Any suggestions?

Reply:
Was this resolved?

Reply:
Nope. If you could point me to where this file should live, I'd gladly move it.